### PR TITLE
Document Git Dependency and Change Wording

### PR DESCRIPTION
**Pull request by omidima**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ This package isn't published on pub.dev, to use it, add a git dependency to your
 dependencies:
 ...
 
-  instagram_private_api: any # publish version branch 
+  instagram_private_api: 
+    git:
+      url: https://github.com/Nerixyz/instagram_private_api.git # master branch
```
Comment (Nerixyz) [inline README.md:17]: Maybe we should also show that you can lock the dependency to a branch/tag/commit:
```yaml
ref: 123ab5f # (optional) branch, tag or commit
```

File: README.md [hunk 2]
```diff
@@ -92,4 +94,4 @@ class FileStateStorage with StateStorage {
 
 
 # Document
-for read document and learn more how about use to this package visit here [instagram-private-api](https://github.com/Nerixyz/instagram_private_api).
\ No newline at end of file
+for read document and learn more how about use to this package visit here [instagram-private-api](https://github.com/Nerixyz/instagram_private_api).
```
Comment (Nerixyz) [inline README.md:97]:
```suggestion
For documentation visit the GitHub repository [instagram-private-api](https://github.com/Nerixyz/instagram_private_api).
```

File: README.md [hunk 3]
```diff
@@ -92,4 +94,4 @@ class FileStateStorage with StateStorage {
 
 
 # Document
```
Comment (Nerixyz) [inline README.md:96]:
```suggestion
# Documentation
```

**Review by Nerixyz — CHANGES_REQUESTED** [review]
The last changes are really from your first PR but I didn't see them.

**Comment by omidima** [comment]
solved problem

**Comment by omidima** [comment]
solved problem

**Review by Nerixyz — APPROVED** [review]

**Pull request merged**